docs/guide/snaps-development-guide.md

```diff
@@ -307,6 +304,21 @@ Follow these instructions to inspect the background process and view its console
   - Click on "Details"
   - Under "Inspect Views", click `background.html`
 
+### Publishing Your Snap
+
+Snaps are npm packages, so publishing a Snap is as simple as publishing an npm package.
+Refer to the [npm cli documentation](https://docs.npmjs.com/cli/v8/commands/npm-publish) for details on publishing to the public registry.
+Take note of the following details specific to Snaps:
+
+- The version in `package.json` and `snap.manifest.json` must match
+- The image specified in `iconPath` in the manifest file will be used as the icon displayed when installing and displaying confirmations from the Snap
+
+After publishing the Snap, any dapp can connect to the Snap by using the snapId `npm:[packageName]`.
```

Comment:
Perhaps we should include an example here?

Reply:
Will have to do that another time.